cmd/install/install.go

```diff
@@ -229,3 +241,25 @@ func downloadArgoCD(osPlatform string) error {
 	}
 	return nil
 }
+
+func installCAPI(ver string) error {
+	c, err := client.New("")
+	if err != nil {
+		return err
+	}
+	options := client.InitOptions{
+		Kubeconfig:              client.Kubeconfig{Path: kubeconfigPath},
+		CoreProvider:            ver,
+		BootstrapProviders:      nil,
```

Comment:
What good does installing CAPI do if no bootstrap and infrastructure providers are installed? In practice, a user will want to use a set of providers right? What if they want to add providers after this initial installation.

Reply:
That's a totally sound idea. Will add it

Reply:
@bcle I tried adding additional non default infrastructure providers. And seems every such provider has some prerequisite which varies according to the provider.
For instance installing AWS components gives me this error:
```shell
ubuntu@ip-172-31-25-241:~/arlon$ sudo arlon install --infrastructure aws,gcp --bootstrap talos,kubeadm --kubeconfigPath /home/ubuntu/.kube/config
Note: SUDO access is required to install the required tools for Arlon

✓ kubectl is already present at default(/usr/local/bin/kubectl)location or user specifed location

✓ git is already present in the path

✓ argocd is already present at default(/usr/local/bin/argocd)location or user specifed location

Attempting to install CAPI...
Error: failed to get provider components for the "aws" provider: failed to perform variable substitution: value for variables [AWS_B64ENCODED_CREDENTIALS] is not set. Please set the value using os environment variables or the clusterctl config file
```

Reply:
The current workaround for this is to execute the command like so:
```shell
ubuntu@ip-172-31-25-241:~/arlon$ sudo AWS_B64ENCODED_CREDENTIALS=<the credential goes here> arlon install --infrastructure aws --bootstrap talos,kubeadm --kubeconfigPath /home/ubuntu/.kube/config
Note: SUDO access is required to install the required tools for Arlon

✓ kubectl is already present at default(/usr/local/bin/kubectl)location or user specifed location

✓ git is already present in the path

✓ argocd is already present at default(/usr/local/bin/argocd)location or user specifed location

Attempting to install cluster-api:v1.1.5 with infrastructure providers [aws] and bootstrap providers [talos kubeadm]✓ CAPI is installed...
```